script/barrelize.js

```diff
@@ -42,6 +42,9 @@ const ignoredTypes = [
   'BackgroundReplacementVideoFrameProcessorDelegate',
   'BackgroundReplacementFilter',
 
+  // These are classes we don't want to export
+  'StatsCollector',
```

Comment:
We need to export `ClientMetricReport` due to `metricDidReceive(clientMetricReport: ClientMetricReport)`.